source/Octopus.Tentacle/Commands/ShowConfigurationCommand.cs

```diff
@@ -86,26 +87,14 @@ async Task StartAsync()
 
         internal async Task CollectConfigurationSettings(DictionaryKeyValueStore outputStore)
         {
-            var configStore = instanceSelector.Current.Configuration;
+            homeConfiguration.Value.WriteTo(outputStore);
 
-            var oldHomeConfiguration = new HomeConfiguration(ApplicationName.Tentacle, configStore!, instanceSelector);
-            var homeConfiguration = new WritableHomeConfiguration(ApplicationName.Tentacle, outputStore, instanceSelector);
-            homeConfiguration.SetHomeDirectory(oldHomeConfiguration.HomeDirectory);
+            tentacleConfiguration.Value.WriteTo(outputStore,
+                TentacleConfiguration.CertificateSettingName,
```

Comment:
These are excluded values - it's not super clear in github though. Might change this.

Reply:
Fixed